### DISCUSSION TITLE
VPN crashes and my IP is left exposed. What's the best solution?

### DISCUSSION THREAD

**Discussion by donelson52** [transmission/transmission#2738] [Q&A]
VPN crashes and my IP is left exposed. What's the best solution?

Is there a way to pause all Transmission when my IP changes?

Thank you

**Comment by thanhminhmr** [comment]
I think using a firewall is a valid solution. Simply block all out-going connections except ones that connect to the VPN server and voila! (But beware that some other things like DNS and NTP may not work correctly.)
